monkestation/code/modules/storytellers/storytellers/mystic.dm

```diff
@@ -1,6 +1,7 @@
 /datum/storyteller/mystic
 	name = "The Mystic"
-	desc = "The Mystic gives events from beyond the vale, some of which may even be magic in nature."
+	desc = "The Mystic gives events from beyond the veil, some of which may even be magic in nature."
 	tag_multipliers = list(TAG_SPOOKY = 1.2, TAG_MAGICAL = 1.5, TAG_SPACE = 1.1)
 	weight = 2
+	restricted = TRUE
```

Comment:
unrestrict the storyteller but reduce the weight of `/datum/round_event_control/summon_wizard_event` down to 4 from 8

Reply:
storyteller was unrestricted, wizard events removed as per host request

Reply:
![image](https://github.com/Monkestation/Monkestation2.0/assets/78111117/35b91535-9ed2-4852-b10f-286f88132b41)